libbeat/docs/shared-ssl-config.asciidoc

```diff
@@ -211,157 +339,234 @@ key: |
-| ECDHE-RSA-AES-128-GCM-SHA256 | TLS 1.2 only.
-| ECDHE-RSA-AES-256-CBC-SHA |
-| ECDHE-RSA-AES-256-GCM-SHA384 | TLS 1.2 only.
-| ECDHE-RSA-CHACHA20-POLY1205 | TLS 1.2 only.
-| ECDHE-RSA-RC4-128-SHA | Disabled by default. RC4 not recommended.
-| RSA-3DES-CBC3-SHA |
-| RSA-AES-128-CBC-SHA |
-| RSA-AES-128-CBC-SHA256 | TLS 1.2 only. Disabled by default.
-| RSA-AES-128-GCM-SHA256 | TLS 1.2 only.
-| RSA-AES-256-CBC-SHA |
-| RSA-AES-256-GCM-SHA384 | TLS 1.2 only.
-| RSA-RC4-128-SHA | Disabled by default. RC4 not recommended.
-|===
+[float]
+[[server-certificate-authorities]]
+==== `certificate_authorities`
 
-Here is a list of acronyms used in defining the cipher suites:
+The list of root certificates for server verifications is only required if 
+`client_authentication` is configured. If `certificate_authorities` is empty or not set, and
+`client_authentication` is not configured, the system keystore is used.
 
-* 3DES:
-  Cipher suites using triple DES
+If `certificate_authorities` is self-signed, the host system needs to trust that CA cert as well.
+By default you can specify a list of files that +{beatname_lc}+ will read, but you can also embed a certificate
+directly in the `YAML` configuration:
 
-* AES-128/256:
-  Cipher suites using AES with 128/256-bit keys.
+[source,yaml]
+----
+certificate_authorities:
+  - |
+    -----BEGIN CERTIFICATE-----
+    MIIDCjCCAfKgAwIBAgITJ706Mu2wJlKckpIvkWxEHvEyijANBgkqhkiG9w0BAQsF
+    ADAUMRIwEAYDVQQDDAlsb2NhbGhvc3QwIBcNMTkwNzIyMTkyOTA0WhgPMjExOTA2
+    MjgxOTI5MDRaMBQxEjAQBgNVBAMMCWxvY2FsaG9zdDCCASIwDQYJKoZIhvcNAQEB
+    BQADggEPADCCAQoCggEBANce58Y/JykI58iyOXpxGfw0/gMvF0hUQAcUrSMxEO6n
+    fZRA49b4OV4SwWmA3395uL2eB2NB8y8qdQ9muXUdPBWE4l9rMZ6gmfu90N5B5uEl
+    94NcfBfYOKi1fJQ9i7WKhTjlRkMCgBkWPkUokvBZFRt8RtF7zI77BSEorHGQCk9t
+    /D7BS0GJyfVEhftbWcFEAG3VRcoMhF7kUzYwp+qESoriFRYLeDWv68ZOvG7eoWnP
+    PsvZStEVEimjvK5NSESEQa9xWyJOmlOKXhkdymtcUd/nXnx6UTCFgnkgzSdTWV41
+    CI6B6aJ9svCTI2QuoIq2HxX/ix7OvW1huVmcyHVxyUECAwEAAaNTMFEwHQYDVR0O
+    BBYEFPwN1OceFGm9v6ux8G+DZ3TUDYxqMB8GA1UdIwQYMBaAFPwN1OceFGm9v6ux
+    8G+DZ3TUDYxqMA8GA1UdEwEB/wQFMAMBAf8wDQYJKoZIhvcNAQELBQADggEBAG5D
+    874A4YI7YUwOVsVAdbWtgp1d0zKcPRR+r2OdSbTAV5/gcS3jgBJ3i1BN34JuDVFw
+    3DeJSYT3nxy2Y56lLnxDeF8CUTUtVQx3CuGkRg1ouGAHpO/6OqOhwLLorEmxi7tA
+    H2O8mtT0poX5AnOAhzVy7QW0D/k4WaoLyckM5hUa6RtvgvLxOwA0U+VGurCDoctu
+    8F4QOgTAWyh8EZIwaKCliFRSynDpv3JTUwtfZkxo6K6nce1RhCWFAsMvDZL8Dgc0
+    yvgJ38BRsFOtkRuAGSf6ZUwTO8JJRRIFnpUzXflAnGivK9M13D5GEQMmIl6U9Pvk
+    sxSmbIUfc2SGJGCJD4I=
+    -----END CERTIFICATE-----
+----
 
-* CBC:
-  Cipher using Cipher Block Chaining as block cipher mode.
+[float]
+[[server-certificate]]
+==== `certificate: "/etc/pki/server/cert.pem"`
 
-* ECDHE:
-  Cipher suites using Elliptic Curve Diffie-Hellman (DH) ephemeral key exchange.
+The path to the certificate for SSL authentication is required. If the certificate
+is not specified, server authentication is not available. 
```

Comment:
'server authentication' is a MUST for TLS. Startup should fail if the file is missing.